src/store/normalize-data.js

```diff
@@ -38,7 +40,9 @@ export const createInitialPipelineState = () => ({
       task: 'Nodes',
       parameters: 'Parameters',
     },
-    disabled: {},
+    disabled: {
+      parameters: isTest ? false : true,
```

Comment:
I am against this set up mainly because this essentially causes great seperation of the app that was loaded in the test environment and the actual set up ( with parameters disabled by default) and in turn the tests that was subsequently tested in all tests suites cannot be used to identify other breaking points in the app - @bru5 Rashida mentioned this was proposed as a quick fix to fix the existing tests, but I do advise strongly against this. 

I had a look at the current failling tests and they should be able to be fixed with a bit of refactor, which should be the right way to approach this given that the refactor is needed anyways as parameters off will be the default going onwards.